deploy/manifests/kip/overlays/mac-tests/provider.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+cloud:
+  aws:
+    vpcID: <FILL-IN>
+    region: <FILL-IN>
+    subnetID: <FILL-IN>
+    accessKeyID: <FILL-IN>
+    secretAccessKey: <FILL-IN>
+etcd:
+  internal:
+    dataDir: /opt/kip/data
+cells:
+  defaultInstanceType: "t3.nano"
+  defaultVolumeSize: "600G"
+  defaultIAMPermissions: "kip-mac-ssm"
```

Comment:
Is this an IAM instance profile you created? Let's add a comment that links to https://docs.aws.amazon.com/systems-manager/latest/userguide/setup-instance-profile.html to at least document how to recreate it.

Reply:
Yes, that's basically minimum kip permissions + permissions required for using CloudStore + AmazonSSMManagedInstanceCore (for debugging purposes). I've added a note about it to the overlay README